vagrant/config.yaml

```
@@ -1,5 +1,5 @@
 admin_password: admin
-version: latest
+version: v2.4.8
```

It makes sense to me to rename to `rancher_version`.

```suggestion
rancher_version: v2.4.8
```

---

All usages of this `version` field in the `Vagrantfile` (I think only one or two, honestly) would need to be altered, too... unsure if it's necessary but if the change is proven to be stable, happy to include it.

---

Cool, done.